src/components/ec-user-info/ec-user-info.vue

```diff
@@ -29,29 +29,24 @@
   </div>
 </template>
 
-<script>
-export default {
-  name: 'EcUserInfo',
-  props: {
-    user: {
-      type: Object,
-      default: () => ({}),
-      required: true,
-    },
-    isCollapsed: {
-      type: Boolean,
-    },
-    isCollapsable: {
-      type: Boolean,
-    },
+<script setup>
+defineProps({
+  user: {
+    type: Object,
+    default: () => ({}),
   },
-  emits: ['toggle'],
-  methods: {
-    toggle() {
-      this.$emit('toggle');
-    },
+  isCollapsed: {
+    type: Boolean,
   },
-};
+  isCollapsable: {
+    type: Boolean,
+  },
+});
+const emit = defineEmits(['toggle']);
+
+function toggle() {
+  emit('toggle');
+}
```

Comment:
CAN BE MERGED: This function can be even inlined into the template because it's so small.

```
@click="emit('toggle')"
```

It was a good practise to create a method in Vue 2 because they weren't cached and were forcing unnecessary component updates, but now, in Vue 3, compiler handles inline functions and caches them automatically.

Reply:
done